dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -729,6 +729,7 @@ export const Explorer = ({
             }));
           });
           history.replace(`/event_analytics/explorer/${res['objectId']}`);
+          return res;
```

Comment:
where is this used?

Reply:
It is not used, and will be used later when we also allow query to be saved to panels. For now just want to keep it consistent with saving visualizations.